### PR TITLE
codecov basic 

### PR DESCRIPTION
**Pull request by sookeke**
- [ ]  codecov for Medical Portal UI
- [ ]  codecov for Medical Portal API
- [ ]  codecov for CMS Adapter
- [ ]  codecov for ICBC Adapter

The only codecov working atm is the Med Portal UI because the repo contains a unit test that is all PASSED. Once other adapter unit test and integration test PASSED the result will be uploaded to codeCov

Its is important to note that codecov uses the default branch for dashboard stat monitor. I recommended change the default branch from main to dev. Currently the default branch is main that is why I have to add anpother push event in the ci pipeline. Pull request hardly occurs in main branch. if we have dev as default branch then we can remove the push event in the ci

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+[![ci-cms-adapter](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml/badge.svg)](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml)
+[![ci-cms-adapter](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml/badge.svg)](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml)
+[![ci-cms-adapter](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml/badge.svg)](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml)
+[![ci-cms-adapter](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml/badge.svg)](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml)
+[![ci-cms-adapter](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml/badge.svg)](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml)
+[![ci-cms-adapter](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml/badge.svg)](https://github.com/bcgov/rsbc-dmf/actions/workflows/ci-cms-adapter.yml)
```
Comment (GeorgeWalker) [inline README.md:10]: There seems to be several duplicate lines here

Comment (GeorgeWalker) [inline README.md:10]: There seems to be several duplicate lines here

File: medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.Development.json [hunk 2]
```diff
@@ -8,5 +8,6 @@
         "Microsoft.Hosting.Lifetime": "Information"
       }
     }
-  }
+  },
+  "cms": "https://rsbc-dfp-cms-adapter-dev.apps.silver.devops.gov.bc.ca/"
```
Comment (GeorgeWalker) [inline medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.Development.json:12]: We've been keeping this type of configuration out of git, can you remove this please?

Comment (GeorgeWalker) [inline medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.Development.json:12]: We've been keeping this type of configuration out of git, can you remove this please?

File: medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.json [hunk 3]
```diff
@@ -9,8 +9,10 @@
       }
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "cms": "https://rsbc-dfp-cms-adapter-dev.apps.silver.devops.gov.bc.ca/"
+
```
Comment (GeorgeWalker) [inline medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.json:14]: We've been keeping this type of configuration out of git, can you remove this please?

Comment (sookeke) [reply medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.json:14]: @GeorgeWalker if you want something out of git, is it not better we add those in the .gitignore file?

File: medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.Development.json (hunk 2 above)
Comment (sookeke) [reply medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.Development.json:12]: application.json out of git?

**Pull request merged**